### PR TITLE
Add test for CASSANDRA-10134

### PR DESCRIPTION
**Pull request by thobbs**
This should be merged after https://issues.apache.org/jira/browse/CASSANDRA-10134 is committed.

### CODE REVIEW

File: replace_address_test.py [hunk 1]
```diff
@@ -228,6 +230,46 @@ def replace_first_boot_test(self):
         debug(movedTokensList[0])
         self.assertEqual(len(movedTokensList), numNodes)
 
+    def fail_without_replace_test(self):
+        """
+        When starting a node from a clean slate with the same address as
```
Comment (mambocab) [inline replace_address_test.py]: Could you please add `@jira_ticket CASSANDRA-10134` to this docstring?

File: replace_address_test.py [hunk 2]
```diff
@@ -228,6 +230,46 @@ def replace_first_boot_test(self):
...
+        for auto_bootstrap in (True, False):
+            debug("Stopping node 3.")
+            node3.stop(gently=False)
+
+            # completely delete the data, commitlog, and saved caches
+            if os.path.exists(os.path.join(node3.get_path(), 'data')):
+                rmtree(os.path.join(node3.get_path(), 'data'))
+
+            node3.set_configuration_options(values={'auto_bootstrap': auto_bootstrap})
+            debug("Starting node 3 with auto_boostrap = %s" % auto_bootstrap)
```
Comment (mambocab) [inline replace_address_test.py]: Could you please use `.format` for string formatting here?

**Comment by mambocab** [comment]
Other than my two small comments, this is ready to go as far as I'm concerned.

**Comment by thobbs** [comment]
Your comments should be addressed now, thanks!

**Comment by mambocab** [comment]
Beautiful, thank you. Ping us when the ticket is closed, and we'll get this merged!

**Comment by thobbs** [comment]
These changes were merged in elsewhere, closing.

**Pull request closed without merging**